### ISSUE TITLE
ODELocalSensitivityProblem

### ISSUE THREAD

**Issue by ayadlin** [JuliaDiffEq/DifferentialEquations.jl#440]
as per request - I am raising the issue:

ODELocalSensitivityProblem fails when running ODEFunction with mass matrix, where (21 equations, 16 differential 6 algebraic)

Best,

A

**Comment by ChrisRackauckas** [comment]
Yes, we should throw an error when we encounter a mass matrix so that way someone doesn't get confused by the probably incorrect solution. The equation needs to be somewhat modified to handle non-trivial mass matrices.
